h5c/vic/src/main/locale/en_US/com_vmware_vic.properties

```diff
@@ -199,6 +199,20 @@ vch.registry.expires=expires
 vch.registry.replace=Replace
 vch.registry.select=Select
 
+# vch operations
+vch.operations.title=Operations User
+vch.operations.provideInfo=Provide vSphere user credential for the VCH to use when interacting with vSphere
+vch.operations.specifyInfo=Specify a non-administrative account to limit permissions for day-to-day VCH operations.
+vch.operations.vSphereUserCredentials=vSphere user credentials
+vch.operations.userNotEmpty=Operations user cannot be empty
+vch.operations.passNotEmpty=Operations user password cannot be empty
+vch.operations.grantPermissions=Grant this user any necessary permissions
+vch.operations.grantUserPermissions=Grant user necessary permissions
+vch.operations.yes=Yes
+vch.operations.no=No
+vch.operations.passwordPlaceholder=Password
```

Comment:
In order to not have a huge amount of specific i18n entries..anything very general ('Yes', 'No', 'Password') should be made generic..maybe something like 'common.label.yes'